english/freedom

```diff
@@ -0,0 +1,7 @@
+The free bird leaps
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.